Replicator/Replicator.cc

```diff
@@ -1101,7 +1092,7 @@ namespace litecore { namespace repl {
         if (!checkpointID)
             return;
 
-        auto [collectionIn, errMsg] = checkCollectionOfMsg(*request, kNotCollectionIndex);
+        auto [collectionIn, errMsg] = checkCollectionOfMsg(*request);
```

Comment:
For some reason, I don't see collectionIn is used.

Reply:
Good catch. It will cause gcc to think it's unused error. I will make sure of it.